src/components/CourseCard.js

```diff
@@ -72,6 +73,7 @@ CourseCard.propTypes = {
   slug: PropTypes.string.isRequired,
   lectureId: PropTypes.number,
   courseViewMode: PropTypes.bool,
+  tags: PropTypes.array,
```

Comment:
I think it is better to use `arrayOf` instead of `array` here.
```
PropTypes.arrayOf(PropTypes.shape({
    tagName: PropTypes.string
))
```

Reply:
Sure. It make sense. I changed type props.